cortx-monitorQuickstartGuide.md

```diff
@@ -103,57 +72,119 @@ This guide provides a step-by-step walkthrough for getting you CORTX-Monitor Ser
 
 2. Build RPMs
     
-   1. Switch to the directory where you've cloned cortx-monitor and run the command:
+   1. Checkout the main branch
    
-        `$ jenkins/build.sh`  
+        `docker run --rm -v /root/cortx:/cortx-workspace ghcr.io/seagate/cortx-build:centos-7.8.2003 make checkout BRANCH=main`  
     
     :page_with_curl: **Notes:** 
     
-    - It takes 2-3 minutes to generate RPMs.
-    - Once the RPMs are successfully generated, you can view it from:
+    - If you get below error it may be because that your git version may be more thatn 2.x. Cortx requires 1.8.3.1. In that case
+      if go to `/root/cortx/cortx-monitor/` and run `git checkout main` 
+        ```
+        error: pathspec 'main' did not match any file(s) known to git.
+        make: *** [checkout] Error 1
+        ```
+
+   2. Build cortx-monitor RPMs
+        ```
+        docker run --rm -v /var/artifacts:/var/artifacts -v /root/cortx:/cortx-workspace ghcr.io/seagate/cortx-build:centos-7.8.2003 make clean cortx-prvsnr cortx-monitor cortx-prereq release_build
+        ```
+        
+        If you check the location `/var/artifacts/0/`, you will see that Cortx RPMs and 3rd party packages are generated.
+
+        ```shell
+        ll /var/artifacts/0/
+        The system output displays as follows:
+
+        [root@ssc-vm-2699 ~]# ll /var/artifacts/0/
```

Comment:
>root@ssc-vm-2699 

remove these hostnames from examples too, we are not supposed to expose any such internal host/domain names

Reply:
Done.